### ISSUE TITLE
Not starting on Arch Linux

### ISSUE THREAD

**Issue by lnicola** [fman-community/fman#446]
~~Sorry for the formatting,~~ I had to get this using `strace`, since it otherwise doesn't get printed to the console:

```
[pid 20606] write(2, "Traceback (most recent call last):
  File \"fbs_runtime/signal_.py\", line 2, in <module>
  File \"/root/dev/fman/venv/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py\", line 714, in load_module
ModuleNotFoundError: No module named 'PyQt5.sip'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File \"fman/main.py\", line 1, in <module>
  File \"/root/dev/fman/venv/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py\", line 631, in exec_module
  File \"fman/__init__.py\", line 1, in <module>
  File \"/root/dev/fman/venv/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py\", line 631, in exec_module
  File \"fman/clipboard.py\", line 3, in <module>
  File \"/root/dev/fman/venv/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py\", line 631, in exec_module
  File \"fman/impl/util/__init__.py\", line 1, in <module>
  File \"/root/dev/fman/venv/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py\", line 631, in exec_module
  File \"fbs_runtime/application_context.py\", line 2, in <module>
  File \"/root/dev/fman/venv/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py\", line 631, in exec_module
  File \"fbs_runtime/signal_.py\", line 4, in <module>
ModuleNotFoundError: No module named 'PySide2'
", 1346) = 1346
```

I do have them installed, but maybe `fman` tries to bundle its own?

```
Python 3.6.6 (default, Jun 27 2018, 13:11:40) 
[GCC 8.1.1 20180531] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import PyQt5
>>> import PyQt5.sip
>>> import PySide2
>>>
```

**Comment by mherrmann** [comment]
Thank you for reporting this. I'm looking into it...

**Comment by mherrmann** [comment]
I just released [fman 1.1.3](https://fman.io/changelog#v1.1.3) with an attempt at a fix. Can you [update](https://fman.io/docs/automatic-updates) and let me know if it works for you now?

**Comment by lnicola** [comment]
It's working now, thanks. Can you also add a TODO or something for the missing console messages?

Oh, okay, nevermind:

```
(/opt/fman/fman "$@" > /dev/null 2>&1 &)
```
